server/tests/user-signup-test.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-undef */
+import chaiHttp from "chai-http";
+import chai from "chai";
+import app from "../server";
+
+chai.should();
+chai.use(chaiHttp);
+const { expect } = chai;
+
+describe("User sign up", () => {
+  it("Should return a 201 and confirm user a valid input", done => {
+    /**
+     * request new user input
+     */
```

Comment:
No need for these comments. Your code should be self-explanatory.

Reply:
Okay